### ISSUE TITLE
Resizing Block Volumes

### ISSUE THREAD

**Issue by barakmich** [coreos/torus#177]
Files can already be resized and appended to; this is just a nice enhancement for the block volume driver. The admin can then run resize2fs, and, magically, it works.

Another great starter bug. Touches one package (block) but uses concepts from the root torus package as well.

**Comment by beh9540** [comment]
@barakmich would this be to just increase the size of the volume, or allow resizing up and down? I have most of increasing done, I think decreasing could be supported, I'm just not sure if we can do it while the volume is mounted.

**Comment by barakmich** [comment]
Yeah, that makes sense. Truncate() will decrease size too -- doing any
resize operation, however, is probably worth doing when not mounted (ie, it
takes the lock)
